lib/grape-swagger/doc_methods/parse_params.rb

```diff
@@ -21,7 +21,7 @@ def call(param, settings, path, route, definitions, consumes) # rubocop:disable
           # optional properties
           document_description(settings)
           document_type_and_format(settings, data_type)
-          document_array_param(value_type, definitions) if value_type[:is_array]
+          document_array_param(value_type, definitions, consumes) if value_type[:is_array]
```

Comment:
🤔 … is it related to this PR, thought it was introduced #927

Reply:
it is related to this PR (handling for `in[:query]` for get requests but it is indeed something that was missed in #927